code/game/objects/machinery/overwatch.dm

```diff
@@ -629,6 +641,7 @@ GLOBAL_LIST_EMPTY(active_cas_targets)
 		H.set_frequency(new_squad.radio_freq)
 
 	transfer_marine.hud_set_job()
+	to_chat(usr, span_boldnotice("[transfer_marine] has been transfered from squad '[old_squad]' to squad '[new_squad]'. Logging to enlistment file."))
 	visible_message(span_boldnotice("[transfer_marine] has been transfered from squad '[old_squad]' to squad '[new_squad]'. Logging to enlistment file."))
```

Comment:
This sends double messages to people standing next to it

Reply:
Hrmmm, should it check for whether the user is a silicon before deciding where to send the message?

Reply:
Implemented a check